src/bridge.js

```diff
@@ -12,19 +26,29 @@ exports.bridge = function(port, config) {
 
         controller: {
             onUserQuery: function(queriedUser) {
-                return {}; // auto-provision users with no additonal data
+                return {}; // auto-provision users with no additional data
             },
 
             onEvent: function(request, context) {
                 // events from matrix
                 const event = request.getData();
-                log.debug("Incoming event:", event);
                 if (event.type !== "m.room.message" || !event.content) {
                     return;
                 }
                 log.info(`Matrix-side: ${event.sender}: ${event.content.body}
                 RoomID: ${event.room_id}, EventID: ${event.event_id} 
                 `);
+                log.info('sending mail');
+                sendMail({
+                    from: 'room+email_localhost@matrix.org',
+                    to: 'user@localhost',
+                    subject: `You have a message from ${event.sender}`,
+                    html: `${event.content.body}`,
+                }, function(err, reply) {
+                    console.log(err && err.stack);
+                    console.dir(reply);
+                });
+                log.info('mail sent');
```

Comment:
Hmm, are you sure? It looks like `sendMail` is a callback thing, so this line should appear in the callback?